### PR TITLE
"Edit Listing|Information" -> "Edit Product Page"

### PR DESCRIPTION
**Pull request by eviljeff**
quick fix for #9857

### CODE REVIEW

**Comment by diox** [comment]
Nota: don't forget to run the l10n extraction script after making that change (we haven't really ran it recently cause it wasn't needed)

**Comment by eviljeff** [comment]
> Nota: don't forget to run the l10n extraction script after making that change (we haven't really ran it recently cause it wasn't needed)

I don't follow.  How is this different commit than any other change to the templates, so requires the l10n extraction script to be specifically run?

**Comment by diox** [comment]
It's not any different, I haven't committed any changes to templates recently so I haven't run it, but realized with this change that we should :)

File: src/olympia/devhub/templates/devhub/addons/listing/item_actions_theme.html [hunk 1]
```diff
@@ -4,7 +4,7 @@ <h5>{{ _('Actions') }}</h5>
     <li>
       <a href="{{ addon.get_dev_url() }}" class="tooltip"
          title="{{ _("Edit details about this theme's listing.") }}">
```
Comment (wagnerand) [inline src/olympia/devhub/templates/devhub/addons/listing/item_actions_theme.html]: Can you update the tooltip here too, please?

Comment (eviljeff) [reply src/olympia/devhub/templates/devhub/addons/listing/item_actions_theme.html]: https://github.com/mozilla/addons-server/pull/9924/commits/dc25ec385d52cde50abd032d068270f3e63196b7

**Pull request merged**